lib/pci.py

```diff
@@ -39,6 +41,152 @@ def get_domains():
         return list(set(domains))
     return []
 
+def  is_nvmf():
+    """
+    Verify if nvmf is configured
+
+    :return: True if nvmf is configure, False if nvmf is not configured
+    """
+
+    try:
+        nvme_list_subsys_output_bytes = subprocess.check_output(['nvme', 'list-subsys'])
+        nvme_list_subsys_output = nvme_list_subsys_output_bytes.decode('utf-8')
+        if "fc" in nvme_list_subsys_output:
+            return True
+        else:
+            return False
+
+    except subprocess.CalledProcessError as e:
+        print(f"Error running 'nvme list-subsys': {e}")
+        return False
+
+def is_rhel8():
+    """
```

Comment:
generic api's will go to helper.py   and pci specific apis will be in pci.py

Reply:
even there get distro utility in helper lib see if we can enhance same if not can be used directly

Reply:
Done resolved